### DISCUSSION TITLE
Methods

### DISCUSSION THREAD

**Discussion by giscus** [karanpratapsingh/portfolio#28] [Comments]
# Methods

In this tutorial, we will learn about methods in Go

https://www.karanpratapsingh.com/courses/go/methods

**Comment by flyrlabs-rch** [comment]
> We can omit the variable part of the receiver as well if we're not using it
>
> `func (Car) UpdateName(...) {}`

This is a poor example because UpdateName always needs access to Car in order to update the name

Actually, finding an example where one did not need a reference to the receiver is a challenge, at least for me

Cheers

**Reply by karanpratapsingh** [reply]
I agree, but I wanted to show the syntax regardless.
